example_test.py

```diff
@@ -72,12 +73,6 @@ def test_create_file(self):
 
     def test_delete_file(self):
         """Test example.delete_file()
-
-        `self.fs.CreateFile()` is convenient because it automatically creates
-        directories in the fake file system and allows you to specify the file
-        contents.
-
-        You could also use `open()` or `file()`.
```

Comment:
Since `file()` has been eliminated in Python 3, let's no longer mention it in places like this.  Of course that is precisely what you have already done.